test/test_modules.py

```diff
@@ -108,6 +111,68 @@ def test_factory_kwargs(self, device, dtype, module_info):
                             buffer.dtype, dtype,
                             f'Buffer {name} is of dtype {buffer.dtype} instead of the expected dtype {dtype}')
 
+    @modules(module_db)
+    def test_check_inplace(self, device, dtype, module_info):
+        # Check if the inplace variant of the module gives the same result as the out of place.
+        module_cls = module_info.module_cls
+        if 'inplace' not in signature(module_cls).parameters:
```

Comment:
I think you could move this into the decorator above like:
```
@modules([module_info for module_info in module_db if 'inplace' in signature(module_info.module_cls)])
```

That way, tests are only generated for modules with that have `inplace` in their signature.